src/modules/Hosts/Hosts/Strings/en-us/Resources.resw

```diff
@@ -239,7 +239,7 @@
     <comment>"Hosts" refers to the system hosts file, do not loc</comment>
   </data>
   <data name="Hosts.[using:Microsoft.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">
-    <value>Seperate multiple hosts by space (e.g. server server.local). Maximum 9 hosts per entry.</value>
+    <value>Separate multiple hosts by space (e.g. server server.local). Maximum 9 hosts per entry.</value>
```

Comment:
Can we remove this here too

https://github.com/microsoft/PowerToys/blob/main/.github/actions/spell-check/expect.txt#L1680

Reply:
yes - I will do that in its own PR so that each PR addresses just one issue